src/app/properties/properties.module.ts

```diff
@@ -36,6 +40,8 @@ import { MatInputModule } from '@angular/material/input';
     AsyncPipe,
     FormsModule,
     ReactiveFormsModule,
+    MatSelectModule,
+    MatCheckboxModule,
```

Comment:
Do not import Angular Material modules by themselves, import Material Module.
```suggestion
    MaterialModule
```